tests/integration/launcher_blueprint/test_solver_versions.py

```diff
@@ -14,66 +14,60 @@
 from starlette.testclient import TestClient
 
 
-class TestSolverVersions:
-    """
-    The purpose of this unit test is to check the `/v1/launcher/versions` endpoint.
-    """
+def test_get_solver_versions(
```

Comment:
I just removed the unneeded class, please ignore whitespaces diffs when reviewing